randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -4852,7 +4872,14 @@ Extra - asset_id: collision_camera_081
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:door_shield_plasma_000
   * Extra - right_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   > Door to Freezer (Power)
-      Screw Attack
+      Any of the following:
+          Screw Attack
+          All of the following:
+              # https://www.youtube.com/watch?v=BuXn_Mkep20
+              Speed Booster Upgrade ≥ 3 and Shinesink Clip (Intermediate) and Can SSC
+              Any of the following:
+                  Speed Booster Upgrade ≥ 3
+                  Plasma Beam and Disabled Door Lock Randomizer
```

Comment:
As this is, that plasma req will never be seen since you have the speed upgrades in the "All"

Reply:
yeah, I just forgot to delete the SB+3 when i added the vanilla doors requirements oops. anyways, going to close this one and redo it when may merges the tricks.